tests/functional/lti_certification/v13/core/test_bad_payloads.py

```diff
@@ -10,29 +10,28 @@ class TestBadPayloads:
     """
 
     def test_no_kid_sent_in_jwt_header(
-        self, jwt_headers, make_jwt, test_payload, do_lti_launch
+        self, jwt_headers, make_jwt, test_payload, do_lti_launch, caplog
     ):
         """
         The KID is missing from the header of the JWT (preventing the verification of the signing of the JWT)
         """
         del jwt_headers["kid"]
 
-        response = do_lti_launch(
-            {"id_token": make_jwt(test_payload, jwt_headers)}, status=403
-        )
+        do_lti_launch({"id_token": make_jwt(test_payload, jwt_headers)}, status=403)
 
-        assert response.html
+        assert "Error decoding JWT. No `kid` in header" in caplog.messages
```

Comment:
A lot of this tests result in a 403 but the cause it's not clear.

I'm adding some logging for both making the test tighter and, once it's time to pass the certification, have something to paste in the "logs" textarea.